bind/motor-mobile/bind.go

```diff
@@ -214,22 +213,76 @@ func IssuePayment(buf []byte) ([]byte, error) {
 // Stat returns general information about the Motor node its wallet and accompanying Account.
 func Stat() ([]byte, error) {
 	if instance == nil {
-		return nil, errWalletNotExists
+		return nil, ct.ErrMotorWalletNotInitialized
 	}
 
 	doc := instance.GetDIDDocument()
 	if doc == nil {
-		return nil, errWalletNotExists
+		return nil, ct.ErrMotorWalletNotInitialized
 	}
-	diddoc, err := types.NewDIDDocumentFromPkg(doc)
+	didDoc, err := rt.NewDIDDocumentFromPkg(doc)
 	if err != nil {
 		return nil, err
 	}
 
 	resp := mt.StatResponse{
 		Address:     instance.GetAddress(),
 		Balance:     int32(instance.GetBalance()),
-		DidDocument: diddoc,
+		DidDocument: didDoc,
+	}
+	return resp.Marshal()
+}
+
+func BuyAlias(buf []byte) ([]byte, error) {
+	if instance == nil {
+		return nil, ct.ErrMotorWalletNotInitialized
+	}
+
+	var request rt.MsgBuyAlias
+	if err := request.Unmarshal(buf); err != nil {
+		return nil, fmt.Errorf("unmarshal request: %s", err)
+	}
+
+	resp, err := instance.BuyAlias(request)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp.Marshal()
```

Comment:
This is still needed yes?

Reply:
Yes, it now relies on the implementation in pkg/motor